<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcomeDialog</class>
 <widget class="QDialog" name="welcomeDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>452</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>538</width>
    <height>452</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>538</width>
    <height>452</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Welcome to wikiLYNX</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextBrowser" name="appLogo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>75</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>71</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#000000&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/base/images/wikiLYNX_logo.svg&quot; width=&quot;90&quot; height=&quot;70&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
       <property name="openLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier Prime</family>
         <pointsize>48</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>wikiLYNX</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label0">
     <property name="font">
      <font>
       <family>Noto Sans</family>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Welcome to wikiLYNX!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabPlay">
      <attribute name="icon">
       <iconset theme="home"/>
      </attribute>
      <attribute name="title">
       <string>Play</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="playerName">
         <property name="font">
          <font>
           <family>Courier Prime</family>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Enter your name</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Who art thou?</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="0,0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>12</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>81</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Difficulty: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="refreshButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>21</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>21</width>
               <height>21</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>21</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="refresh"/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="chk">
             <property name="maximumSize">
              <size>
               <width>221</width>
               <height>21</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Courier Prime</family>
               <stylestrategy>PreferDefault</stylestrategy>
              </font>
             </property>
             <property name="toolTip">
              <string>Number of checkpoints</string>
             </property>
             <property name="text">
              <string>chk</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="passcodeInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>221</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>221</width>
               <height>21</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Courier Prime</family>
               <stylestrategy>PreferDefault</stylestrategy>
              </font>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Select Level</string>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::InsertPolicy::InsertAlphabetically</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::SizeAdjustPolicy::AdjustToMinimumContentsLengthWithIcon</enum>
             </property>
             <property name="placeholderText">
              <string>Select Level</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>81</width>
               <height>21</height>
              </size>
             </property>
             <property name="text">
              <string>Checkpoints: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>21</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Noto Sans</family>
               <pointsize>10</pointsize>
               <stylestrategy>PreferDefault</stylestrategy>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Level: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="difficulty">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>221</width>
               <height>21</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>221</width>
               <height>21</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Courier Prime</family>
               <stylestrategy>PreferDefault</stylestrategy>
              </font>
             </property>
             <property name="toolTip">
              <string>Current Level difficulty</string>
             </property>
             <property name="text">
              <string>difficulty</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_playFooter">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Add Custom Levels in Settings!</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="initButton">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Courier Prime</family>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="toolTip">
            <string>Start Game</string>
           </property>
           <property name="text">
            <string>INIT!</string>
           </property>
           <property name="icon">
            <iconset theme="play"/>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSettings">
      <attribute name="icon">
       <iconset theme="settings"/>
      </attribute>
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="allowSitesToggle">
         <property name="text">
          <string>Allow other sites outside Wikipedia (Access to reference links) </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="killToggle">
         <property name="text">
          <string>Allow changing game window during game session</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Icon Theme: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Font: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFontComboBox" name="fontComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_settingsButtons">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="editLevelButton">
           <property name="maximumSize">
            <size>
             <width>94</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Add your own custom levels</string>
           </property>
           <property name="text">
            <string>Edit Levels</string>
           </property>
           <property name="icon">
            <iconset theme="edit"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sLogsButton">
           <property name="maximumSize">
            <size>
             <width>94</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Show Logs folder</string>
           </property>
           <property name="text">
            <string>Show Logs</string>
           </property>
           <property name="icon">
            <iconset theme="history"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cLogsButton">
           <property name="maximumSize">
            <size>
             <width>94</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Clear Logs</string>
           </property>
           <property name="text">
            <string>Clear Logs</string>
           </property>
           <property name="icon">
            <iconset theme="delete"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTextBrowser" name="privateLogo">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>61</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::InputMethodHint::ImhNone</set>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;#161616&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/base/images/DAWN.svg&quot; width=&quot;100&quot; height=&quot;50&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextInteractionFlag::NoTextInteraction</set>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newsButton">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>View Leaderboard</string>
       </property>
       <property name="text">
        <string>News</string>
       </property>
       <property name="icon">
        <iconset theme="news"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="statsButton">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>View Leaderboard</string>
       </property>
       <property name="text">
        <string>Stats</string>
       </property>
       <property name="icon">
        <iconset theme="trophy"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>WhatsThisCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Learn how to play</string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
       <property name="icon">
        <iconset theme="help"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>WhatsThisCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>About wikiLYNX</string>
       </property>
       <property name="text">
        <string>About</string>
       </property>
       <property name="icon">
        <iconset theme="info"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="statusIndicator">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="pending"/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>28</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Courier Prime</family>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string>status</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Version: 1.5.0 (Stable)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
